.github/workflows/ci.yml

```diff
@@ -40,10 +40,6 @@ jobs:
             - name: Install dependencies 👨🏻‍💻
               run: |
                 make install
-                make update-peer-deps
-                make build-scss-pkg
-                make symlink
-                make build-vue-pkg
                 make symlink
```

Comment:
I would think we still need it for the same reason we did before... so that the versions of things in the repo are used, rather than the currently published versions.